crawl-ref/source/dat/database/de/insult.txt

```diff
@@ -959,13 +971,13 @@ insult general
 
 @insult_adjective1@es @insult_adjective2@es Gepäck
 
-@insult_adjective1@er @insult_adjective2@er Bandersnitch
+@insult_adjective1@er @insult_adjective2@er Schnatterrind
 
 @insult_adjective1@er @insult_adjective2@er Seepferdchen
 
 @insult_adjective1@er @insult_adjective2@er Bettler
 
-@insult_adjective1@er @insult_adjective2@er Blabberskit
+@insult_adjective1@er @insult_adjective2@er Plaudertasche
```

Comment:
It's die Plaudertasche, right? In which case, the adjectives should have -e instead of -er.